tests/foreman/cli/test_contenthost.py

```diff
@@ -451,71 +384,69 @@ class TestCHKatelloAgent(CLITestCase):
 
     org = None
     env = None
-    cv = None
+    content_view = None
     activation_key = None
-    org_is_set_up = False
 
-    def setUp(self):
-        """Create VM, subscribe it to satellite-tools repo, install katello-ca
-        and katello-agent packages
+    @classmethod
+    def setUpClass(cls):
+        """Create Org, Lifecycle Environment, Content View, Activation key
```

Comment:
The old setUp() method is split into a setUpClass() method and a new setUp() method.  The steps which are not required to run for each test are now moved to the setUpClass() to speed up the test execution.